src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimSpecCallbackHandler.java

```diff
@@ -150,8 +166,14 @@ private CallbackResponse validateUnavailableDates(CallbackParams callbackParams)
         // This will be taken care via different story,
         // because we don't have AC around this date field validation in ROC-9455
         CaseData caseData = callbackParams.getCaseData();
-        SmallClaimHearing smallClaimHearing = caseData.getRespondent1DQ().getRespondent1DQHearingSmallClaim();
-        List<String> errors = unavailableDateValidator.validateSmallClaimsHearing(smallClaimHearing);
+        List<String> errors;
+        if ("SMALL_CLAIM".equals(caseData.getResponseClaimTrack())) {
```

Comment:
Same comment here, Constants interface might be better way.

Reply:
this is not sonarqube compliant
https://rules.sonarsource.com/java/RSPEC-1214

so did this
```
public final class Status {             // Compliant
   public static final int OPEN = 1;
   public static final int CLOSED = 2;
}
```